shared/tuebingen/verkehr.tex

```diff
@@ -40,6 +40,7 @@ \subsubsection*{Mit dem Bus}
 
 \subsubsection*{Mit dem Fahrrad}
 Wie in den meisten Unistädten wird in Tübingen sehr viel Fahrrad gefahren. Es gibt daher auch vor allen wichtigen Gebäuden die Möglichkeit sein Fahrrad anzuschließen. Darüber kann das Fahrrad im Bus die steilen Anstiege mitgenommen werden (zumindest wenn nicht gerade Sperrzeit ist, in vielen Bussen befindet sich dazu ein Aushang).
+\footnote{Spinde für Fahrrad- und Motorrad-Kleidung gibt es auf der Morgenstelle am Übergang von Hörsaalzentrum zum B-Bau, sowie auf dem Sand im Raum C119.}
```

Comment:
Sorry, mein letzter Kommentar war vlt. missverständlich. Der `Stand dd.mm.yy` Teil wäre für mich eine Fußnote. Der normale Text kann selbstverständlich in den normalen Fließtext.